packages/terra-site/src/examples/table/NoStripedTable.jsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import Table from 'terra-table';
+
+const shortid = require('shortid');
```

Comment:
We are generating the keys in example files only, we are not generating the keys in the code.